lms/djangoapps/ora_staff_grader/urls.py

```diff
@@ -13,4 +13,7 @@
     path(
         'initialize', InitializeView.as_view(), name='initialize'
     ),
+    path(
+        'fetch-submission', SubmissionFetchView.as_view(), name='fetch-submission'
+    )
```

Comment:
This deviates from contract. Should be the path below (or argue/update contract docs).

```suggestion
    path(
        'submission', SubmissionFetchView.as_view(), name='fetch-submission'
    )
```